version/version.go

```diff
@@ -26,4 +26,7 @@ var (
 	// Revision is filled with the VCS (e.g. git) revision being used to build
 	// the program at linking time.
 	Revision = ""
+
+	// GoVersion is filled at linking time
+	GoVersion = ""
```

Comment:
what about using `runtime.Version` directly? it seems we don't need to pass it from Makefile. The go builder will help us.

Reply:
hmm; maybe makes sense--want to handle in a followup PR?; looks like this is already merged :)

Reply:
will do!

Reply:
@fuweid  i can take it if you do not mind :)